vale/Grafana/AllowsTo.yml

```
@@ -7,3 +7,4 @@ action:
   name: replace
 swap:
   allows to: allows you to
+  allows to: makes it possible to
```

I think the current patch results in only "makes it possible to" being proposed:

> Did you mean 'makes it possible to' instead of 'allows to'?

With the following suggestion:

```suggestion
  allows to: allows you to|makes it possible to
```

You instead get the output:

> Did you mean 'allows you to' or 'makes it possible to' instead of 'allows to'?

---

Thank you. Was just reading documentation about how to do this. I figured it was a `|`. :)